### PR TITLE
Add React hooks ESLint rules

### PR DESCRIPTION
**Pull request by victmo**
Added React hooks ESLint rules and updated package peer deps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fiverr/eslint-config-fiverr",
   "moduleName": "eslint-config-fiverr",
-  "version": "2.0.1",
+  "version": "3.0.0-rc.0",
```
Comment (michael5r) [inline package.json]: Why would this be a major update?

Comment (sahariko) [inline package.json]: No need to bump a major version, please bump a minor instead.

File: rules/react.js [hunk 2]
```diff
@@ -50,6 +51,8 @@ module.exports = {
             "beforeSelfClosing": "never"
         }],
         "react/jsx-uses-vars": 2,
-        "react/jsx-wrap-multilines": 2
+        "react/jsx-wrap-multilines": 2,
+        "react-hooks/rules-of-hooks": 2,
+        "react-hooks/exhaustive-deps": 1
```
Comment (sahariko) [inline rules/react.js]: Please change this to be an error as well.

**Comment by victmo** [comment]
Thanks @sahariko, meant to do this earlier but got caught up with AND CO stuff.
It's already merged but isn't adding new `peerDependencies` a breaking change?

**Comment by sahariko** [comment]
> Thanks @sahariko, meant to do this earlier but got caught up with AND CO stuff.
> It's already merged but isn't adding new `peerDependencies` a breaking change?

Good point 👍

**Pull request merged**